docs/Troubleshooting.md

```diff
@@ -124,11 +124,7 @@ More information on Node debugging can be found [here](https://nodejs.org/api/de
 
 ## Debugging in WebStorm
 
-The easiest way to debug Jest tests in [WebStorm](https://www.jetbrains.com/webstorm/) is using `Jest run/debug configuration`. It will launch tests and automatically attach debugger.
-
-In the WebStorm menu `Run` select `Edit Configurations...`. Then click `+` and select `Jest`. Optionally specify the Jest configuration file, additional options, and environment variables. Save the configuration, put the breakpoints in the code, then click the green debug icon to start debugging.
-
-If you are using Facebook's [`create-react-app`](https://github.com/facebookincubator/create-react-app), in the Jest run/debug configuration specify the path to the `react-scripts` package in the Jest package field and add `--env=jsdom` to the Jest options field.
```

Comment:
we don't need this note about CRA anymore?

Reply:
Yes, it's working fine out of the box without extra manual configuration.
https://youtrack.jetbrains.com/issue/WEB-28401#focus=Comments-27-3087386.0-0

Reply:
wonderful!